contracts/test/RFQ.t.sol

```diff
@@ -311,6 +325,13 @@ contract RFQTest is StrategySharedSetup {
         makerMakerAsset.assertChange(-int256(order.makerAssetAmount));
     }
 
+    function testFillDAIToUSDT_EOAUserAndEOAMaker_MakerOnlyApproveRFQ() public {
+        RFQLibEIP712.Order memory order = DEFAULT_ORDER;
+        // maker only approve RFQ
+        _makerOnlyApproveRFQ(order.makerAddr, order.makerAssetAddr);
+        testFillDAIToUSDT_EOAUserAndEOAMaker();
```

Comment:
Not sure calling another test case is a good idea. I think it can be independent since the rest can be done in few lines.

Reply:
Also, I think we can focus on allowance/settlement here so no need to test each combination of EOA/Contract signing case.